### PR TITLE
create a halls page profile #5

### PR DESCRIPTION
**Pull request by Samaamro20**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "test": "tape src/tests/test.js",
     "lint": "eslint ./src",
     "lint-fix": "eslint --fix ./src",
-    "eslint-init": "eslint --init ./src"
+    "eslint-init": "eslint --init ./src",
+    "db:query": "psql -U postgres -h localhost -d hmcqc -a -f ./query.sql"
```
Comment (shahenazmonia) [inline package.json]: There's a problem during running this command 
```psql -U  "database-user"  "database-name" -a -f ./query.sql```

File: src/backEnd/controllers/profile.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      serviceDetails.findAll({
+        where: { userId: user_id },
+        attributes: ['id', 'name', 'location', 'price', 'image_url'],
+        raw: true,
+        include: [{
+          model: user,
+        }],
+      })
+        .then((result) => {
+          console.log('rees', result);
```
Comment (shahenazmonia) [inline src/backEnd/controllers/profile.js]: remove console log

File: src/views/profile.hbs [hunk 3]
```diff
@@ -1,5 +1,25 @@
 {{>header}}
+
+{{#each result}}
+
+Name: {{user.name}}
```
Comment (shahenazmonia) [inline src/views/profile.hbs]: name here is hidden, if you consoling the result you would see the value has stored in string ```{{'user.name'}}```

File: package.json (hunk 1 above)
Comment (Samaamro20) [reply package.json]: Done

File: src/backEnd/controllers/profile.js (hunk 2 above)
Comment (Samaamro20) [reply src/backEnd/controllers/profile.js]: Done

File: src/views/profile.hbs (hunk 3 above)
Comment (Samaamro20) [reply src/views/profile.hbs]: Done Thanks ^_^

**Pull request merged**